sdk/monitor/azure-monitor-opentelemetry/azure/monitor/opentelemetry/_autoinstrumentation/configurator.py

```diff
@@ -26,6 +33,17 @@ def _configure(self, **kwargs):
         if not _is_attach_enabled():
             warn(_PREVIEW_ENTRY_POINT_WARNING)
         try:
+            if environ.get(OTEL_TRACES_EXPORTER, "").lower().strip() != "none":
+                kwargs.setdefault("trace_exporter_names", ["azure-monitor-opentelemetry-exporter"])
+                try:
+                    sample_rate = float(environ.get("OTEL_TRACES_SAMPLER_ARG"))
```

Comment:
Just curious, is supporting `OTEL_TRACES_SAMPLER_ARG` in auto-instrumentation new functionality? I don't seem to see the logic previously. Then again, autoinstrumentation is only used by attach currently right?

Reply:
No. It was previously on the otel side. We just provided the entry point name and the base init components method initialized it with the sampler arg.